pkg/pillar/dpcreconciler/linuxitems/adapter.go

```diff
@@ -150,6 +150,15 @@ func (c *AdapterConfigurator) Create(ctx context.Context, item depgraph.Item) er
 		c.Log.Error(err)
 		return err
 	}
+	// Make sure the ethernet interface is DOWN before renaming
```

Comment:
If the interface is currently up, shouldn't we bring it back up after the reanme+MAC change?
(But keep it down it if it currently down.)

Reply:
We bring it back up few lines later with `netlink.LinkSetUp(kernLink)`
Regardless of the original interface state, we need it to be UP when this methods ends.

Reply:
Ok